applications/nrf_desktop/doc/ble_scan.rst

```diff
@@ -134,8 +134,8 @@ a. The scanning is stopped and the |NCS|'s :ref:`nrf_bt_scan_readme` automatical
 #. After the :ref:`nrf_desktop_ble_discovery` completes the peer discovery, the :ref:`nrf_desktop_ble_conn_params` receives the ``ble_discovery_complete_event`` and updates the Bluetooth connection interval.
 
 .. important::
-   If a Bluetooth peer is already connected with a 1-ms connection interval, the next peer is connected with a 10-ms connection interval instead of 7.5 ms.
-   The peer is connected with a 10-ms connection interval also in case :kconfig:`CONFIG_BT_MAX_CONN` is set to value greater than 2 and :kconfig:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option is enabled.
+   If a Bluetooth peer is already connected with a 1 ms connection interval, the next peer is connected with a 10-ms connection interval instead of 7.5 ms.
```

Comment:
So in this line, because the "ms" are followed by the word "connection", they are not considered modifiers and therefor do not need a hyphen? Just want to make sure I understand correctly.

Reply:
Oh wait. In the conversation thread for the `applications/nrf_desktop/README.rst` doc, it seems that "ms" is a modifier in these cases, so it should have a hyphen? 
Will add them now, and edit out later if needed.